server/server/server.go

```diff
@@ -170,8 +169,8 @@ func New(params Params) (*Server, error) {
 	// Init telemetry
 	telemetryID := settings["TelemetryID"]
 	if len(telemetryID) == 0 {
-		telemetryID = uuid.New().String()
-		if err = params.DBStore.SetSystemSetting("TelemetryID", uuid.New().String()); err != nil {
+		telemetryID = appModel.NewID("")
+		if err = params.DBStore.SetSystemSetting("TelemetryID", telemetryID); err != nil {
```

Comment:
This is unrelated but looked like a bug...  the telemetry id that was generated and used was not the same id stored in system settings.